storage/database/partitioned_database.go

```diff
@@ -149,6 +150,98 @@ func (pdb *partitionedDB) Close() {
 	}
 }
 
+type partitionedDBIterator struct {
+	iterators []Iterator
+	key       []byte
+	value     []byte
+
+	//numBatches uint
+	//
+	//taskCh   chan pdbBatchTask
+	//resultCh chan pdbBatchResult
+}
+
+// NewIterator creates a binary-alphabetical iterator over the entire keyspace
+// contained within the key-value database.
+func (pdb *partitionedDB) NewIterator() Iterator {
+	// TODO-Klaytn implement this later.
+	return nil
+}
+
+// NewIteratorWithStart creates a binary-alphabetical iterator over a subset of
+// database content starting at a particular initial key (or after, if it does
+// not exist).
+func (pdb *partitionedDB) NewIteratorWithStart(start []byte) Iterator {
+	iterators := make([]Iterator, 0, pdb.numPartitions)
+	for i := 0; i < int(pdb.numPartitions); i++ {
+		iterators = append(iterators, pdb.partitions[i].NewIteratorWithStart(start))
+	}
+
+	for _, iter := range iterators {
+		if iter != nil {
+			if !iter.Next() {
+				iter = nil
+			}
+		}
+	}
+
+	return &partitionedDBIterator{iterators, nil, nil}
+}
+
+// NewIteratorWithPrefix creates a binary-alphabetical iterator over a subset
+// of database content with a particular key prefix.
+func (pdb *partitionedDB) NewIteratorWithPrefix(prefix []byte) Iterator {
+	// TODO-Klaytn implement this later.
+	return nil
+}
+
+func (pdi *partitionedDBIterator) Next() bool {
+	var minIter Iterator
+	minIdx := -1
+	minKey := []byte{0}
+	minKeyValue := []byte{0}
+
+	for idx, iter := range pdi.iterators {
+		if iter != nil {
+			if bytes.Compare(minKey, iter.Key()) >= 0 {
+				minIdx = idx
+				minIter = iter
+				minKey = iter.Key()
+				minKeyValue = iter.Value()
```

Comment:
These values could be assigned when this `for` loop is finished.

Reply:
This implementation is not used for now. 
It will be implemented later.

For now, bloom filter uses empty memory DB.